app/src/debug/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,24 @@
+<resources>
+    <!--
+    TODO: Before you run your application, you need a Google Maps API key.
+
+    To get one, follow this link, follow the directions and press "Create" at the end:
+
+    https://console.developers.google.com/flows/enableapi?apiid=maps_android_backend&keyType=CLIENT_SIDE_ANDROID&r=C3:D6:C2:CC:3C:A0:0D:D1:8E:EC:B6:8B:8B:3B:DC:AC:6C:F1:CB:79%3Bcom.gusta.wakemehome
+
+    You can also add your credentials to an existing key, using these values:
+
+    Package name:
+    C3:D6:C2:CC:3C:A0:0D:D1:8E:EC:B6:8B:8B:3B:DC:AC:6C:F1:CB:79
+
+    SHA-1 certificate fingerprint:
+    C3:D6:C2:CC:3C:A0:0D:D1:8E:EC:B6:8B:8B:3B:DC:AC:6C:F1:CB:79
+
+    Alternatively, follow the directions here:
+    https://developers.google.com/maps/documentation/android/start#get-key
+
+    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
+    string in this file.
+    -->
+    <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">AIzaSyA5GiLoWGHIyxrfM8EUWqx3_MAHdV3GCvc</string>
+</resources>
```

Comment:
It's related to Google maps for separate env, development and production.
For now it is same thing but for future usages will be able to test with another key to prevent not nessesery api calls